service/frontend/dcRedirectionHandler.go

```diff
@@ -1662,6 +1662,7 @@ func (handler *DCRedirectionHandlerImpl) UpdateWorkflow(
 				return err
 			}
 			resp, err = remoteClient.UpdateWorkflow(ctx, request)
+			return err
```

Comment:
This is unrelated but it _is_ a bug that snuck in just ahead of the staticcheck re-enablement and now doesn't make it past staticcheck